fastlane/Fastfile

```diff
@@ -575,9 +575,9 @@ platform :ios do
             app_name = build.app_center_app_name
         end
 
-        
-        sh "echo SHA of IPA is:"
-        sh "shasum -a 256 ../#{build.artifact_path(with_filename: true)}.ipa || true"
+        sh "SHA256=$(shasum -a 256 ../#{build.artifact_path(with_filename: true)}.ipa || true)"
```

Comment:
are you sure it's the device ipa that we have here and not the simulator one?

Reply:
this is an artifact that we use to upload to appcenter. 
There are 2 approaches:
- get sha256 from this artifact
- or wait a result from appcenter, download ipa, and compute the sha256.
The first option sounds more efficient.

Reply:
sounds good! I guess I got confused as in upload_s3 we upload 2 ipas in some cases one for simulator (QA) and one for device, so i wasn't sure it was the right one here